api/node.go

```diff
@@ -70,7 +70,7 @@ func (s *Server) NodeUpdate(rw http.ResponseWriter, req *http.Request) error {
 	// Only scheduling disabled node can be evicted
 	// Can not enable scheduling on an evicting node
 	if n.EvictionRequested == true && n.AllowScheduling != false {
-		return errors.Wrap(err, "Need to disable scheduling on this node for node eviction, or cancel eviction to enable scheduling on this node.")
+		return errors.New("need to disable scheduling on this node for node eviction, or cancel eviction to enable scheduling on this node")
```

Comment:
Use `fmt.Errorf`, and add node/disk name to the error message.

Reply:
Sure, will fix it. Thanks.